include/ibamr/IBFEInstrumentPanel.h

```diff
@@ -0,0 +1,153 @@
+
```

Comment:
Please use the standard file format from the rest of the library for header files. See, e.g., [IBInstrumentPanel.h](https://github.com/IBAMR/IBAMR/blob/master/include/ibamr/IBInstrumentPanel.h) for an example.